shell/src/main/java/alluxio/shell/command/ReportCommand.java

```diff
@@ -42,7 +44,7 @@ public String getCommandName() {
 
   @Override
   void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioException, IOException {
-    LineageFileSystem.get().reportLostFile(path);
+    LineageFileSystem.get(FileSystemContext.INSTANCE, LineageContext.INSTANCE).reportLostFile(path);
```

Comment:
yea, the `LineageFileSystem` should come through the constructor, just like the current `FileSystem`